apps/sensenet/src/components/content-list/reference-field.tsx

```diff
@@ -21,6 +21,10 @@ const useStyles = makeStyles(() => {
       marginLeft: '1em',
       textTransform: 'lowercase',
     },
+    link: {
+      marginLeft: '1em',
```

Comment:
marginRight should be on the icon, because this margin will be there even if no icon rendered